storage/api/Storage.Samples.Tests/BucketWebsiteConfigutationTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Xunit;
+
+[Collection(nameof(BucketFixture))]
+public class BucketWebsiteConfigutationTest
+{
+    private readonly BucketFixture _bucketFixture;
+
+    public BucketWebsiteConfigutationTest(BucketFixture bucketFixture)
+    {
+        _bucketFixture = bucketFixture;
+    }
+
+    [Fact]
+    public void BucketWebsiteConfigutation()
+    {
+        BucketWebsiteConfigutation bucketWebsite = new BucketWebsiteConfigutation();
```

Comment:
```suggestion
        BucketWebsiteConfigutation bucketWebsiteConfigutation = new BucketWebsiteConfigutation();
```

General Note:
In my pervious PR what I maintain the standard:
BucketWebsiteConfigutationSample  <-- Class name
BucketWebsiteConfigutation <--method name
BucketWebsiteConfigutation.cs  <-- file name
After all it's depends on reviewer, let's see it they gives this changes or not but I would re-commanded the same.